extension/healthcheckextension/config.go

```diff
@@ -37,6 +38,11 @@ type Config struct {
 	// The default endpoint is "0.0.0.0:13133".
 	TCPAddr confignet.TCPAddr `mapstructure:",squash"`
 
+	// Path represents the path the health check service will
+	// serve.
```

Comment:
Move this to the line above, doesn't need to be a seperate line :)

Reply:
done